db/osqlsession.c

```diff
@@ -130,7 +130,13 @@ int osql_sess_close(osql_sess_t **psess, int is_linked)
 
     if (is_linked) {
```

Comment:
we need to ignore is_linked if sockbplog

Reply:
FTR, doing so by checking sees->impl->socket, which also needs to be set at creation.